expected/17.0/extra/returning.out

```diff
@@ -0,0 +1,519 @@
+--SET log_min_messages  TO DEBUG4;
+--SET client_min_messages  TO DEBUG4;
+--Testcase 01:
+CREATE EXTENSION sqlite_fdw;
+--Testcase 02:
+CREATE SERVER sqlite_svr FOREIGN DATA WRAPPER sqlite_fdw
+OPTIONS (database '/tmp/sqlite_fdw_test/common.db');
+--Testcase 03:
+CREATE SERVER sqlite2 FOREIGN DATA WRAPPER sqlite_fdw;
+--Testcase 04:
+IMPORT FOREIGN SCHEMA main FROM SERVER sqlite_svr INTO public;
+--Testcase 05:
+SELECT * FROM "type_STRING";
+  col   
+--------
+ string
+ TYPE
+ type
+(3 rows)
+
+--Testcase 06:
+ EXPLAIN (VERBOSE, COSTS OFF)
+UPDATE "type_STRING" SET col = '_' || substr(col, 2) RETURNING *;
+                             QUERY PLAN                              
+---------------------------------------------------------------------
+ Update on public."type_STRING"
+   Output: col
+   ->  Foreign Scan on public."type_STRING"
+         Output: ('_'::text || substr(col, 2)), col, "type_STRING".*
+         SQLite query: SELECT `col` FROM main."type_STRING"
+(5 rows)
+
+--Testcase 07:
+UPDATE "type_STRING" SET col = '_' || substr(col, 2) RETURNING *;
+  col   
+--------
+ _tring
+ _YPE
+ _ype
+(3 rows)
+
+--Testcase 08:
+SELECT * FROM "type_STRING";
+  col   
+--------
+ _tring
+ _YPE
+ _ype
+(3 rows)
+
+--Testcase 09:
+ EXPLAIN (VERBOSE, COSTS OFF)
+DELETE FROM "type_STRING" RETURNING *;
+WARNING:  DELETE ... RETURNING doesn't return values (not implemented)
+                         QUERY PLAN                         
+------------------------------------------------------------
+ Delete on public."type_STRING"
+   Output: col
+   ->  Foreign Scan on public."type_STRING"
+         Output: col
+         SQLite query: SELECT `col` FROM main."type_STRING"
+(5 rows)
+
+--Testcase 10:
+DELETE FROM "type_STRING" RETURNING *;
+WARNING:  DELETE ... RETURNING doesn't return values (not implemented)
+ col 
+-----
+ 
+ 
+ 
+(3 rows)
+
+--Testcase 11:
+ EXPLAIN (VERBOSE, COSTS OFF)
+INSERT INTO "type_STRING"(col) VALUES ('string') ON CONFLICT DO NOTHING RETURNING *;
```

Comment:
Currently, SQLiteFDW does not support ON CONFLICT DO NOTHING clause, I think you should separate this feature to another Pull Request or remove this its test and implementation.

Reply:
@lamdn1409 , why this is a problem? I thinks this is just `INSERT` function tested and small internal detail or this touch some other code layers?